src/DistributionApi.php

```diff
@@ -62,14 +65,16 @@ public function __construct(
         ModuleDataProvider $moduleDataProvider,
         ShopDataProvider $shopDataProvider,
         string $prestashopVersion,
-        string $downloadDirectory
+        string $downloadDirectory,
+        int $shouldAddShopUrl
```

Comment:
Now that I think of it, `Configuration->get()` will likely return `(string)'1'` that is casted into int here. Would you mind testing if it works if we change this to (bool) ?

Reply:
it works :)